test/algorithm/group.hpp

```diff
@@ -0,0 +1,26 @@
+//          Copyright Odin Holmes 2016.
+// Distributed under the Boost Software License, Version 1.0.
+//    (See accompanying file LICENSE.md or copy at
+//          http://www.boost.org/LICENSE_1_0.txt)
+#pragma once
+
+#include <kvasir/mpl/algorithm/group.hpp>
+#include <kvasir/mpl/algorithm/make_sequence.hpp>
+
+namespace {
+	struct group_test {
+		group_test() {
```

Comment:
this should be static constexpr at least to avoid binary overhead
it might be possible that this function will always create a linker reference which could create problems for people including the tests in embedded environments.

Reply:
Normally, there is no linker reference in anonymous namespace.
Note: this is a ctor